examples/tf_job.yaml

```diff
@@ -21,4 +21,9 @@ spec:
               name: tensorflow
           restartPolicy: OnFailure
     - replicas: 2
-      tfReplicaType: PS
\ No newline at end of file
+      tfReplicaType: PS
```

Comment:
This is a regression due to the refactor we should be supplying a default pod spec. Can you fix that or open up an issue to fix that?

Reply:
I will file an issue for it :-)